src/schema/posts.ts

```diff
@@ -1109,7 +1109,7 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       const partialPost = await ctx.con.getRepository(Post).findOneOrFail({
         select: ['id', 'sourceId', 'private'],
         relations: ['source'],
-        where: [{ id }, { slug: id }],
+        where: [{ id }, { slug: id }, { visible: true }],
```

Comment:
Not sure on product need, think we allow it to the owner still? (or if bookmarked etc?) (not 100% sure but feels there might be a edge-case)

Reply:
In graphorm we actually do:
```
qb
  .andWhere(`"${alias}"."deleted" = false`)
  .andWhere(`"${alias}"."visible" = true`)
```

so maybe deleted actually instead of banned

Reply:
@capJavert Done, actually cleaned it up so we better re-use same evaluations.